test/Package.js

```diff
@@ -105,15 +105,127 @@ describe("Package", () => {
     });
   });
 
+  describe(".set versionSerializer()", () => {
+    it("should call 'serialize' method of serializer'", () => {
+
+      const mockSerializer = {
+        serialize: jest.fn((pkg) => {
+          return pkg;
+        }),
+        deserialize: jest.fn((pkg) => {
+          return pkg;
+        })
+      };
+
+      pkg.versionSerializer = mockSerializer;
+
+      expect(mockSerializer.deserialize.mock.calls.length).toBe(1);
```

Comment:
Either test or just it's description is wrong. You call `deserialize` method, not `serialize` like the title says.